tests/chainer_tests/functions_tests/pooling_tests/test_spatial_pyramid_pooling_2d.py

```diff
@@ -190,11 +190,6 @@ def check_invalid_poolings(self):
         with self.assertRaises(ValueError):
             functions.spatial_pyramid_pooling_2d(self.v, 3, pooling='avg')
 
-        with testing.assert_warns(DeprecationWarning), \
```

Comment:
I think it makes sense to leave this test.

Reply:
Added the test back.